backend/User/mixins.py

```diff
@@ -111,29 +112,32 @@ def logout_user(self, request: Request) -> Response:
         return Response({"message": "User logged out"}, status=status.HTTP_200_OK)
 
 
-class DeleteUserMixin:
-	"""
-	Mixin to delete a user by ID.
-	"""
-	def delete_user(self, request: Request, user_id: int) -> Response:
-		try:
-			logout(request)
-		except Exception:
-			return Response(status=status.HTTP_400_BAD_REQUEST)
-		result = get_object_or_404(User, pk=user_id)
-		if not isinstance(result, User):
-				return result
-
-		result.username = f"deleted_user_{user_id + 42}"
-		result.set_password(get_random_string(length=30))
-		result.is_active = False
-		result.is_staff = False
-		result.is_superuser = False
-		result.insertTS = None
-		result.last_login = None
-		result.is_online = False
-		result.save()
-		return Response(status=status.HTTP_204_NO_CONTENT)
+class DeleteUserMixin(ChangeDeletedUserTournamentNamesMixin):
+    """
+    Mixin to delete a user by ID.
+    """
+    def delete_user(self, request: Request, user_id: int) -> Response:
+        try:
+            logout(request)
+        except Exception:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+        result = get_object_or_404(User, pk=user_id)
+        if not isinstance(result, User):
+                return result
+
+        result.username = f"deleted_user_{user_id + 42}"
+        result.set_password(get_random_string(length=30))
+        result.is_active = False
+        result.is_staff = False
+        result.is_superuser = False
+        result.insertTS = None
```

Comment:
Why 🐫 instead of 🐍?

Reply:
I remember that that field had been changed ages ago. I'll fix it now.